src/js/plugin/sdk/services/MarathonService.js

```diff
@@ -0,0 +1,22 @@
+
+import config from "../config/config";
+import ajaxWrapper from "../../../helpers/ajaxWrapper";
+
+export default class MarathonService {
+
+  request(opts = {}) {
```

Comment:
Let's make request static so that you don't need to create an instance of `MarathonService`.

Reply:
:+1: